dolphinscheduler-bom/pom.xml

```diff
@@ -380,6 +381,13 @@
                 <scope>test</scope>
             </dependency>
 
+            <dependency>
+                <groupId>com.dameng</groupId>
+                <artifactId>DmJdbcDriver18</artifactId>
+                <version>${dm-jdbc.version}</version>
+                <scope>test</scope>
```

Comment:
Are you sure we want to use `test` scope for this dependency? Users who want to use dm plugin have to download the jar and put it in the correct folder. I think `DmJdbcDriver18` is Apache license so we can use it safely in our distribution package

Reply:
I checked  DmJdbcDriver18 is Apache License  2.0, I want the user to use directly, without adding the driver jar package. I'll remove the test scope.